docs/source/install.rst

```diff
@@ -4,6 +4,89 @@
 Installing Volttron
 ===================
 
+Installing Linux Virtual Machine
+--------------------------------
+Volttron requires a Linux system to run. For most this will require virtual machine (VM). This section will describe the
+steps necessary to install VOLTTRON using Oracle VirtualBox software. Virtual Box is free and can be
+downloaded from https://www.virtualbox.org/wiki/Downloads.
+
+|VirtualBox Download|
+
+.. |VirtualBox Download| image:: images/vbox-download.png
+
+After installing VirtualBox, it will also be necessary to download a Linux operating system image for use on the VM.
+Ubuntu 14.04 LTS or Linux Mint 17 or later is the recommended Linux operating system for use with VOLTTRON, however any
```

Comment:
VOLTTRON has been tested using Ubuntu 14.04, 16.04; raspian, debian 8,9; Linux Mint 17, 18; and CentOS 7.  However, any modern apt based Linux distribution should work out of the box.